lib/miq_automation_engine/engine/miq_ae_engine.rb

```diff
@@ -305,9 +305,11 @@ def self.resolve_automation_object(uri, user_obj, attr = nil, options = {}, read
 
   def self.ae_user_object(options = {})
     raise "user_id not specified in Automation request" if options[:user_id].blank?
-    # raise "group_id not specified in Automation request" if options[:miq_group_id].blank?
+    # raise "miq_group_id not specified in Automation request" if options[:miq_group_id].blank?
```

Comment:
The `raise` will be re-introduced upstream after this PR is merged.  This will allow the PR to be back-ported and to older branch and give more burn-in time upstream.